.vscode/settings.json

```diff
@@ -2,5 +2,11 @@
   "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
   "editor.tabSize": 2,
   "files.trimTrailingWhitespace": true,
-  "javascript.preferences.importModuleSpecifier": "relative"
+  "javascript.preferences.importModuleSpecifier": "relative",
+  "json.schemas": [
+    {
+      "fileMatch": ["app/manifest/*/*.json"],
```

Comment:
Should we restrict this just to the "base" and "chrome" manifests? It looks like this schema was just written for the Chrome extension manifest, not for Firefox.

Though the schema does seem to allow extra properties, so maybe it works adequately well for Firefox as well :thinking: Offhand I'm not aware of them using the same property for different things.

Reply:
Doesn't matter which way to me, though it is useful in the partial manifest files too, I think.

Reply:
OK, we can try applying to both, and restrict it just to Chrome+base if it causes issues.

I've tested it out, and it looks like there are no errors with the Firefox manifest. Just a few Firefox-specific keys that are missing from the schema.